recipes/xproperty/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D BUILD_TESTS=OFF -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```

Comment:
You can use `clangdev` package which gives you clang 4.0.0 with `libstdc++` as default standard library.